### ISSUE TITLE
Incorrect health check path registered with spring-boot-actuator 1.3

### ISSUE THREAD

**Issue by bpgray** [spring-cloud/spring-cloud-consul#88]
When using spring-boot-actuator 1.3, the health endpoint is now nested under server.contextPath when management.port is set. Note that this is different behavior than with spring-boot-actuator 1.2.5, and potentially breaking. Regardless of the spring-boot-actuator version, spring-cloud-consul does not seem to consider the contextPath of the management endpoints. Considering the appropriate contextPath in ConsulLifecycle.createCheck could be helpful and reduce the need for configuration via properties.

**Comment by spencergibb** [comment]
So, you are asking to automate what is [documented](https://github.com/spring-cloud/spring-cloud-consul/blob/master/docs/src/main/asciidoc/spring-cloud-consul.adoc#http-health-check)?

**Comment by bpgray** [comment]
Yes, it seems that I am asking to automate what is documented in the case of setting management.contextPath or server.contextPath to something other than null. This seems like a reasonable enhancement, given that in most cases we will know the health endpoint location if the contextPath is set, and therefore won't need the user to modify the healthCheckPath in the properties.

**Comment by ezraroi** [comment]
+1
